### ISSUE TITLE
Add funds to contract execution

### ISSUE THREAD

**Issue by iboss-ptk** [spoo-bar/cosmy-wasmy#15]
**Is your feature request related to a problem? Please describe.**
When contract execution requires funds, cosmy wasmy can't add funds to the execution.

**Describe the solution you'd like**
Take [this contract](https://github.com/osmosis-labs/osmosis-rust/blob/70840a85a0f68526b3e904dfabebc42bf314116e/examples/cosmwasm/contracts/osmosis-stargate/src/msg.rs#L11-L15) for example, it sends stargate msg to osmosis' tokenfactory to create denom, and it requires creation fee.

It would be nice if we can add `##funds` as an addition input for contract execution, parsed as coins and pass them to the execute function and strip them out before actually sending the msg. (could just extract and strip it [here](https://github.com/spoo-bar/cosmy-wasmy/blob/b80e8d221071b161b1ea58c7fa8fef8598dbae59/src/helpers/Cosmwasm/API.ts#L71)?)
```json
{
  "create_denom": {
    "subdenom": "poopcoin",
    "initial_mint": "10000000000"
  },
  "##funds": [
    "1000000000uosmo"
  ]
}
```

Happy to contribute if you agree with the approach!

**Comment by spoo-bar** [comment]
Yuupp. I been thinking on how to include option to send funds during msg call.
api.ts, as you linked, is a good place to handle that.

Would love your contribution :)

**Comment by spoo-bar** [comment]
Implemented in the latest release [v2.0.0](https://github.com/spoo-bar/cosmy-wasmy/releases/tag/v2.0.0)

**Comment by spoo-bar** [comment]
Implemented and released in [v2.0.0](https://github.com/spoo-bar/cosmy-wasmy/releases/tag/v2.0.0)
